cedar-wasm/src/lib.rs

```diff
@@ -1,8 +1,252 @@
 #![forbid(unsafe_code)]
 
-use wasm_bindgen::prelude::*;
+use wasm_bindgen::prelude::wasm_bindgen;
+
+use cedar_policy::{
+    Authorizer, Context, Entities, EntityUid, Policy, PolicySet, Request, Schema, ValidationMode,
+    Validator,
+};
+
+use serde_json::json;
+
+use std::collections::HashSet;
+use std::str::FromStr;
 
 #[wasm_bindgen(js_name = "getCedarVersion")]
 pub fn get_cedar_version() -> String {
     std::env!("CEDAR_VERSION").to_string()
 }
+
+#[wasm_bindgen(js_name = "isAuthorized")]
+pub fn is_authorized(
```

Comment:
for isAuthorized I was thinking we would just annotate the version in cedar policy with the  #[wasm_bindgen] macro under feature flag, as well as the input and output structs for it